website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -281,7 +281,7 @@ spec:
     {
       name: 'Name',
       description:
-        "The name of the destination service for all intentions defined in this config entry. This may be set to the wildcard character (`*`) to match all services that don't otherwise have intentions defined.",
+        "The name of the destination service for all intentions defined in this config entry. This may be set to the wildcard character (`*`) to match all services that don't otherwise have intentions defined. Wildcard intentions cannot be used when defining L7 [`Permissions`](/docs/connect/config-entries/service-intentions#permissions).",
```

Comment:
Would it be worthwhile to codify this in validation logic for the config entry write. That way even if you didn't read the docs there would be an error returned by the API indicating the same thing.

Reply:
This seems like a breaking change though so it would probably need to wait for the next major release not to mention having some supporting code for stripping out the Permissions for wildcard entries when they are reinserted into memdb.

Reply:
It already behaved this way: https://github.com/hashicorp/consul-enterprise/blob/main/agent/structs/config_entry_intentions.go#L645

Reply:
Paul is just calling out how it already worked better on the docs.